index.html

```diff
@@ -76,9 +76,44 @@ <h3>Définition de l'accessibilité web</h3>
         </figcaption>
     </figure>
 
-    <h2>Situation de handicap numérique</h2>
+    <h2>Concevoir pour tout le monde</h2>
 
-    <h3>Design inclusif</h3>
+    <h3><strong>Étudier la diversité</strong> pour réduire l'exclusion</h3>
+    <p>Qui sont vos utilisateurs et utilisatrices dans leur diversité ?
+        Assez difficile de savoir exactement, parce que c'est intraçable, mais ce qui est certain, c'est que parmi elleux, certain·es sont en situation de handicap avec :
+    </p>
+    <ul>
+        <li>troubles dys (10 % de la population) ;</li>
+        <li>déficience visuelle (3 %), daltoniens (4 %) ;</li>
+        <li>difficultés auditives (11,5 %) ;</li>
+        <li>difficultés difficultés motrices des membres supérieurs, etc.;</li>
+    </ul>
+    <img src="/handicaps.webp" alt=""/>
+    <p>C'est sans compter les situations de handicap temporaire (bras dans le plâtre, dépression, etc.),
+        situationnel (lieu bruyant, contexte stressant, etc.)
+        ou liées à l'équipement non optimal (écran mal calibré, souris cassée, etc.).
+    </p>
+    <p>
+        <strong>20 à 40 %</strong> des internautes ont un accès difficile, partiel ou impossible aux informations et services en ligne.
+    </p>
+    <p>Questions à se poser en user research :</p>
```

Comment:
```suggestion
    <p>Questions à se poser en User Research :</p>
```
conventionnellement avec majuscules